### DISCUSSION TITLE
THIS SERVER IS NOW INSTALLING SOMETHING, WAIT 5 MIN AND TRY AGAIN

### DISCUSSION THREAD

**Discussion by argon06** [totumonline/totum-ru-issues-and-discussions#566] [Как это сделать?] [answered]
добрый день! Столкнулся с проблемой на первом же шаге. Не проходит первая же проверка
sudo ps aux | grep -i apt | wc -l) -ne 1
Если запустить sudo ps aux | grep -i apt | wc -l то у меня 2. 
Я пробовал sudo killall apt - не помогает (apt: процесс не найден).
Пробовал просто sudo ps aux | grep -i apt, но я не понимаю что с этим дальше делать. 
Вот результат:
root         797  0.0  0.2 422480 11520 ?        Ssl  08:37   0:00 /usr/sbin/thermald --systemd --dbus-enable --adaptive
argon06     4473  0.0  0.0   6572  2304 pts/0    S+   08:58   0:00 grep --color=auto apt

Может кто хоть направление подскажет?

**Comment by ttm-alexeypolunin** [comment, marked as answer]
Непонятно почему это выпадает по поиску `apt` возможно при запуске `thermald` там где-то есть параметры совпадающие с `apt`. 

```
Ssl 08:37 0:00 /usr/sbin/thermald --systemd --dbus-enable --adaptive
```


Тк `thermald` это служба управления температурой, то скорее всего это локальный комп, на серверах я такого не видел ранее.


Удалите из файла установщика проверку — строки `4`-`14`:

```
if [[ $(sudo ps aux | grep -i apt | wc -l) -ne 1 ]]
then
  echo
  echo "THIS SERVER IS NOW INSTALLING SOMETHING, WAIT 5 MIN AND TRY AGAIN"
  echo
  exit 0
else
  echo
  echo "APT is OK..."
  echo
fi
```
